security/acme-client/src/opnsense/scripts/OPNsense/AcmeClient/certhelper.php

```diff
@@ -735,6 +735,7 @@ function run_acme_validation($certObj, $valObj, $acctObj)
             case 'dns_euserv':
                 $proc_env['EUSERV_Username'] = (string)$valObj->dns_euserv_user;
                 $proc_env['EUSERV_Password'] = (string)$valObj->dns_euserv_password;
+                $acme_hook_options[] = "--insecure";
```

Comment:
That's a tough pill to swallow. As a EUserv customer I would try to get them to fix it. But that's just my opinion. :)